src/base/bittorrent/nativesessionextension.cpp

```diff
@@ -30,11 +30,33 @@
 
 #include <libtorrent/alert_types.hpp>
 
+#include <QtGlobal>
```

Comment:
Is it in use?

Reply:
> Is it in use?

It was in the initial implementation. Removed.